Gemfile.lock

```diff
@@ -0,0 +1,52 @@
+GEM
```

Comment:
My understanding is that it should be committed, for the same reason that we recently started committing our `package-lock.json`: https://stackoverflow.com/a/4151540/385997